mirth/docker/docker-compose-mirth.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  mc:
+    image: nextgenhealthcare/connect:4.5.2
+    platform: linux/amd64
+    restart: always
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      - DATABASE=mysql
+      - DATABASE_URL=jdbc:mysql://db:3306/mirthdb
+      - DATABASE_MAX_CONNECTIONS=20
+      - DATABASE_USERNAME=${MYSQL_USER}
+      - DATABASE_PASSWORD=${MYSQL_PASSWORD}
+      - DATABASE_MAX_RETRY=2
+      - DATABASE_RETRY_WAIT=10000
+      - KEYSTORE_STOREPASS=docker_storepass
+      - KEYSTORE_KEYPASS=docker_keypass
+      - VMOPTIONS=-Xmx512m
+    volumes:
+      - ./data/volumes/appdata:/opt/connect/appdata
+      - ./data/volumes/logs:/opt/connect/logs
+    ports:
+      - 8090:8080/tcp
+      - 8443:8443/tcp
+      - ${MIRTH_INBOUND_PORT}:${MIRTH_INBOUND_PORT}
+    depends_on:
+      - db
+  db:
+    image: mysql:5.7.44
```

Comment:
I tried again running MySQL 8, with one configuration change that it said was required.  But it still didn't work, and I didn't dig in too deeply.

Reply:
Just curios, @mogoodrich , what kind of errors you got when running MySQL 8?

Reply:
I've have to look, but @cioan  this was about the Mirth service being unable to connect to MySQL 8 (mysql started fine) so likely about the Mirth application not being compatible with MySQL 8, and I didn't want to get too deep in a rabbit hole with that.